titan/features/haart.py

```diff
@@ -99,18 +101,36 @@ def update_agent(self, model: "model.TITAN"):
                     ]
                     num_haart_agents = self.counts[self.agent.race][self.agent.sex_type]
 
-                    if num_haart_agents < (haart_params.prob * num_dx_agents):
+                    # take value from dictionary for cap
+                    if num_haart_agents < (haart_params.cap * num_dx_agents):
                         self.initiate(model)
                 else:
-                    if model.run_random.random() < (
-                        haart_params.prob * model.calibration.haart.coverage
-                    ):
-                        self.initiate(model)
+                    if self.ever and self.agent.location.params.hiv.reinit_haart:
```

Comment:
why have `haart_cap` and `haart_reinit` in the hiv params, should put those in haart feature params

Reply:
You mean under demographics? I don't see haart feature params anywhere else, should that be a separate file?